cores/cv32e40p/user_manual/source/debug.rst

```diff
@@ -45,21 +45,31 @@ Interface
 +===============================+===========+============================================+
 | ``debug_req_i``               | input     | Request to enter Debug Mode                |
 +-------------------------------+-----------+--------------------------------------------+
+| ``debug_havereset_o``         | output    | Debug status: Core has been reset          |
++-------------------------------+-----------+--------------------------------------------+
+| ``debug_running_o``           | output    | Debug status: Core is running              |
++-------------------------------+-----------+--------------------------------------------+
+| ``debug_halted_o``            | output    | Debug status: Core is halted               |
++-------------------------------+-----------+--------------------------------------------+
 | ``dm_halt_addr_i[31:0]``      | input     | Address for debugger entry                 |
 +-------------------------------+-----------+--------------------------------------------+
 | ``dm_exception_addr_i[31:0]`` | input     | Address for debugger exception entry       |
 +-------------------------------+-----------+--------------------------------------------+
 
 ``debug_req_i`` is the "debug interrupt", issued by the debug module when the core should enter Debug Mode. The ``debug_req_i`` is synchronous to ``clk_i`` and requires a minimum assertion of one clock period to enter Debug Mode. The instruction being decoded during the same cycle that ``debug_req_i`` is first asserted shall not be executed before entering Debug Mode.
 
+``debug_havereset_o`` is used to signal that the CV32E40P has been reset. ``debug_havereset_o`` = 1 during assertion of ``rst_ni``. It will only go to 0 some (not further specified number of) cycles after ``rst_ni`` has been deasserted and ``fetch_enable_i`` has been sampled high.
+
+``debug_running_o`` is used to signal that the CV32E40P is running normally, i.e. it is no longer in its reset state as signaled via ``debug_havereset_o`` nor is it in debug mode as signaled via ``debug_halted_o``.
```

Comment:
Consider removing the example starting from "i.e. it is no longer..."

Reply:
Done

Reply:
Done